flink-cdc-connect/flink-cdc-source-connectors/pom.xml

```diff
@@ -50,4 +50,18 @@ under the License.
         <module>flink-sql-connector-vitess-cdc</module>
     </modules>
 
+    <dependencies>
+        <dependency>
+            <groupId>com.ververica</groupId>
+            <artifactId>flink-cdc-common</artifactId>
+            <version>${revision}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.ververica</groupId>
+            <artifactId>flink-cdc-runtime</artifactId>
+            <version>${revision}</version>
```

Comment:
Should use ${project.version} in dependencies.

Reply:
Done. Thank you for advice.